php-practice/section5.php

```diff
@@ -54,7 +54,9 @@ function getElementFromArray($arr, $index) {
  * Explain in your own words why the above 3 examples of code are effectively
  * the same exact code, written in different ways.
  *
- * ANSWER:
+ * ANSWER: The first example is a function followed by the call parameters while the third (the second being a busted out versino of the third) uses variables 
+ * to create the array and the function before calling it. All three use a function just in a different order, both use "echo" to call the function, and both define
+ * the array so it can be called.
```

Comment:
Good... Some of your details are mixed up but overall you got it!

Here are some clarifications:

1. The first is a busted out version of the 2nd, not the other way around.

2. You said the third "uses variables to create the array and the function before calling it". None of the examples create the function -- the function was only created once (on lines 27-29) but it was **called** in all 3 examples. Whenever you see `getElementFromArray(...)`, that is calling that function. The `echo $message` in example 3 isn't calling the function, it's just echoing out the `return`ed value from that function, which was stored in `$message` above.

I think you already understand most (all?) of this, but I wanted to clarify since I wasn't sure based on some of your wording.